images/wavedrom/c-load_store_pair.adoc

```diff
@@ -0,0 +1,46 @@
+//## Zcmlsd instructions
+
+[wavedrom, ,svg]
+....
+{reg: [
+  {bits: 2, name: 'op',     type: 8, attr: ['2','C2']},
+  {bits: 5, name: 'imm',    type: 5, attr: ['5','offset[4:2|7:6]']},
```

Comment:
Shouldn't the offsets here be scaled up one bit? So `offset[8:3]` instead of `[7:2]`

Reply:
Same applies to `C.SDSP`

Reply:
```suggestion
  {bits: 5, name: 'imm',    type: 5, attr: ['5','offset[4:3|8:6]']},
```